uktrade/data-hub-frontend
# Cleanup of webpack config

## Description of change

After this change, webpack will transpile all files in `node_modules `directory.

This will increase the build time but will also ensure that packages written in ES6 will be compatible with older browsers.

## Test instructions

Try to compile the client-side code in prod mode:
```
NODE_ENV=production npm run build
```

Then run the app and check if everything works as before:
```
NODE_ENV=production node .
```


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
